modules/storages/app/views/storages/admin/storages/new_oauth_client.html.erb

```diff
@@ -7,35 +7,39 @@
 <%= toolbar title: "#{t("storages.provider_types.#{@storage.provider_type}.name")} #{t("storages.label_oauth_client_details")}" %>
 
 <%= labelled_tabular_form_for @oauth_client, url: admin_settings_storage_oauth_client_path do |f| -%>
-  <div class="form--field -required">
-    <%= f.text_field :client_id,
-                     label: t('storages.label_oauth_client_id'),
-                     required: true,
-                     size: 40,
-                     container_class: '-wide' %>
-    <span class="form--field-instructions">
+  <fieldset class="form--fieldset">
```

Comment:
This differs from the structure in the other file, where you hade an additional `form--section`. Please make sure that they are consistent